google/cloud/forseti/common/util/file_uploader.py

```diff
@@ -35,7 +35,7 @@ def upload_json(data, gcs_upload_path):
     """
     try:
         with tempfile.NamedTemporaryFile() as tmp_data:
-            tmp_data.write(parser.json_stringify(data))
+            tmp_data.write(parser.json_stringify(data).encode('utf-8'))
```

Comment:
Should change to `.encode()` since I think by default `.encode()` uses `utf-8` and this fits with how we have done other `.encode()` in the code.